frontend/src/components/common/PanelHeader.vue

```diff
@@ -2,27 +2,25 @@
     <div :style="headerStyle">{{ headerText }}</div>
 </template>
 
-<script lang="ts">
-import { Component, Prop, Vue } from "vue-property-decorator";
+<script lang="ts" setup>
+import { computed } from "vue";
+interface Props {
+    fontSize?: string;
+    fontWeight?: string;
+    padding: string;
+    headerText: string;
+}
 
-@Component
-export default class PanelHeader extends Vue {
-    // Properties
-    @Prop({ default: "medium" })
-    private fontSize!: string;
-    @Prop({ default: "normal" })
-    private fontWeight!: string;
-    @Prop()
-    private padding?: string;
-    @Prop()
-    headerText!: string;
+const props = withDefaults(defineProps<Props>(), {
+    fontSize: "medium",
+    fontWeight: "normal",
+});
 
-    get headerStyle() {
-        let headerStyle = `font-size: ${this.fontSize}; font-weight: ${this.fontWeight};`;
-        if (this.padding) {
-            headerStyle = headerStyle + `padding: ${this.padding}`;
-        }
-        return headerStyle;
+const headerStyle = computed(() => {
+    let headerStyle = `font-size: ${props.fontSize}; font-weight: ${props.fontWeight};`;
+    if (props.padding) {
+        headerStyle = headerStyle + `padding: ${props.padding}`;
```

Comment:
ziehst du den headerstyle hier bitte noch in den String mit rein
```suggestion
        headerStyle = `${headerStyle} padding: ${props.padding}`;
```